.gitlab-ci.yml

```diff
@@ -101,6 +100,7 @@ check_links:
 # Check dependencies
 check_dependencies:
   stage: links
+  allow_failure: true
```

Comment:
is this here to stay?

Reply:
see https://github.com/DIRACGrid/DIRACOS/pull/127#issuecomment-604375728

But I guess this tests, rather than being totally disabled, would need to be adapted